pages/static/serviceworker.js

```diff
@@ -1,3 +1,4 @@
+importScripts('https://storage.googleapis.com/workbox-cdn/releases/4.3.1/workbox-sw.js');
```

Comment:
Can you please have another look if there's no way to reuse the Workbox version amp-sw uses internally? 🙂

Reply:
...and if that's not possible, I'd suggest implementing it without workbox as we don't really need it's features for what we try to accomplish.

Reply:
Workbox is not longer in use now, I've rewritten the `registerRoute()` functions. 👍